mlrun/feature_store/api.py

```diff
@@ -469,19 +481,44 @@ def deploy_ingestion_service(
     function.metadata.project = featureset.metadata.project
     function.metadata.name = function.metadata.name or name
 
-    # todo: add trigger (from source object)
-
     function.spec.graph = featureset.spec.graph
     function.spec.parameters = run_config.parameters
     function.spec.graph_initializer = (
         "mlrun.feature_store.ingestion.featureset_initializer"
     )
     function.verbose = function.verbose or verbose
+    add_source_trigger(source, function)
+
     if run_config.local:
         return function.to_mock_server(namespace=get_caller_globals())
     return function.deploy()
 
 
+def add_source_trigger(source, function):
```

Comment:
i guess this func belongs on a different file (sources or ingestion) not in the API